test/onnx/onnx_test.cpp

```diff
@@ -2556,6 +2556,105 @@ TEST_CASE(loop_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(lpnormalization_double_test)
```

Comment:
Normally there isn't a need to create onnx tests for every supported datatype of the operator unless we have a specific reason to test them. By default, we set the datatype as float.

I would, however, create 2 tests for l1 and l2 norm cases. For the axis, you can specify the attribute in any one of the tests and get code coverage.